### DISCUSSION TITLE
Wiki里面实际使用效果的图裂了

### DISCUSSION THREAD

**Discussion by LilFrOgggg** [chatchat-space/Langchain-Chatchat#3998] [General]
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/89755952/20998b37-8be6-4a04-90fa-ddb293fdb7f1)

**Comment by zRzRzRzRzRzRzR** [comment]
我们之后维护一下，最近太忙了，主要在做新版本还有各自本职工作的事情
